### PR TITLE
Add form inputs for book a trip panel

### PR DESCRIPTION
**Pull request by dreardon1021**
#### What's this PR do?
Creates the book a trip form.
#### Where should the reviewer start?
index.js, index.html, domupdates.js, any styling
#### How should this be manually tested?
The form is currently non functional, however, you can feel how the form will feel when being filled out. 
#### Questions:
- What are your thoughts on the general layout of the site?
- What would you change?

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -59,7 +59,10 @@ function loginUser() {
   $('.login-form').submit(e => {
     e.preventDefault();
   })
-  console.log(tripData)
+  let arrayOfFifty = []
```
Comment (kimalajoy) [inline src/index.js]: Just curious:
What's this doing? I couldn't figure it out!

**Comment by kimalajoy** [comment]
Looks like you've made some really great progress on the project so far! I cloned down your repo to try and check it out your layout and such, but I think I would be able to give you better feedback if you send me the link to the github pages deployed site, if you want some additional feedback.
From what I could see I liked your simple layout and color choices, though!

**Comment by kimalajoy** [comment]
Just kidding, I can't merge it until you set me as contributor!

**Comment by kimalajoy** [comment]
Yay! Merging!

File: src/index.js (hunk 1 above)
Comment (dreardon1021) [reply src/index.js]: So essentially I am generating an array of 50 items. traveler1 -traveler50.

This allows me to check the input value of the username and have something to compare it to. THen I split the string value and just splice out the number part so i can instantiate a new instance of my traveller class

**Pull request merged**